test/test_general_workflow_auth.py

```diff
@@ -69,5 +69,6 @@ def test_general_workflow_auth():
 
     # check storage after deployment
     assert_storage(deploy_info["address"], BALANCE_KEY, "0x14c9")
-    assert_transaction(invoke_tx_hash, "ACCEPTED_ON_L2", expected_signature=SIGNATURE)
```

Comment:
if the signature is not used anywhere else in the original form, perhaps it can already be hexed and just passed like that?